src/v/rp_util/main.cc

```diff
@@ -24,8 +24,7 @@ int corpus_write(char** argv, std::filesystem::path dir) {
     seastar::app_template app;
     try {
         return app.run(1, argv, [dir = std::move(dir)]() -> ss::future<int> {
```

Comment:
nit: you can remove the `ss::future<int>`, I've been doing that because then you can kind of tell (for the most part) where the coroutine lambdas are

Reply:
removed this in next batch of coro fixes